src/status_im2/common/resources.cljs

```diff
@@ -95,3 +101,7 @@
 (defn get-token
   [k]
   (get tokens k))
+
+(defn get-parallax-video
```

Comment:
in theory this is not necessary

you can call parallax video directly

`(parallax-video k)` == `(get-parallax-video k)` is the same

as maps implement https://cljs.github.io/api/cljs.core/IFn

if you want to have a wrapper/exporting a different name, then 
`(def get-parallax-video parallax-video)`
should do the same I believe

Reply:
thanks @cammellos, will check it out :)